frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts

```diff
@@ -1,106 +1,142 @@
+  // Step 1
+  // Select all worker Nodes
+  commonFlows.checkAll().check();
+  commonFlows.checkAll().should('be.checked');
+  // Two dropdowns in the same page.
+  // (Todo: )make dropdown data-test-id be something that can be passed as a prop
+  cy.byLegacyTestID('dropdown-button')
+    .first()
+    .click();
+  cy.byTestDropDownMenu('512Gi').click();
+  wizard.next();
 
-      // Final Step
-      wizard.create();
+  // Step 2
+  if (encrypted) {
+    cy.log('Enabling Encryption');
+    cy.byTestID('encryption-checkbox').click();
+  }
+  wizard.next();
 
-      cy.log('Verify all worker nodes are labelled');
-      cy.exec('oc get nodes -o json').then((res) => {
-        const { items } = JSON.parse(res.stdout);
-        items
-          .map((item) => item.metadata.labels)
-          .filter((item) => item.hasOwnProperty('node-role.kubernetes.io/worker'))
-          .forEach((item) =>
-            expect(item.hasOwnProperty('cluster.ocs.openshift.io/openshift-storage')).toBeTruthy(),
-          );
-      });
+  // Final Step
+  wizard.create();
 
-      // Wait for the storage cluster to reach Ready
-      // Storage Cluster CR flickers so wait for 10 seconds
-      // Disablng until ocs-operator fixes above issue
+  cy.log('Verify all worker nodes are labelled');
+  cy.exec('oc get nodes -o json').then((res) => {
+    const { items } = JSON.parse(res.stdout);
+    items
+      .map((item) => item.metadata.labels)
+      .filter((item) => item.hasOwnProperty('node-role.kubernetes.io/worker'))
+      .forEach((item) =>
+        expect(item.hasOwnProperty('cluster.ocs.openshift.io/openshift-storage')).toBeTruthy(),
+      );
+  });
+
+  // Wait for the storage cluster to reach Ready
+  // Storage Cluster CR flickers so wait for 10 seconds
+  // Disablng until ocs-operator fixes above issue
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(10000);
+  cy.byTestID('resource-status').contains('Ready', { timeout: 900000 });
+};
+
+Cypress.Commands.add('install', (encrypted = false) => {
+  cy.exec(`oc get storagecluster ocs-storagecluster -n ${NS}`, {
+    failOnNonZeroExit: false,
+  }).then(({ code }) => {
+    // Only run Installation if the Storage Cluster doesn't already exist
+    if (code !== 0) {
+      cy.log('Perform OCS Operator installation and StoargeCluster creation');
+      createImagePullSecret(CATALOG.NAMESPACE);
+      createCustomCatalogSource();
+      subscribeToOperator();
+      cy.byTestID('view-installed-operators-btn').click();
+      cy.log('Wait for operator phase to be `Installing`');
+      cy.byLegacyTestID('item-filter').type(`${OCS_OP}`);
+      cy.byTestID('status-text', { timeout: 60000 }).should('have.text', 'Installing');
+      // Waiting for 30 seconds to make sure service accounts are loaded.
       // eslint-disable-next-line cypress/no-unnecessary-waiting
-      cy.wait(10000);
-      cy.byTestID('resource-status').contains('Ready', { timeout: 900000 });
+      cy.wait(30000);
```

Comment:
```suggestion
      cy.wait(30000);
```
Poll for it.

Reply:
Actually. these service accounts are created by the operator.
if we poll it, then we need to hardcode their names and ensure all of them available.
In practice, we will get them instantly in few seconds after operator installation.
The 30 secs is for being on the safer side.